rascal-lsp/src/main/rascal/framework/Rename.rsc

```diff
@@ -310,17 +318,19 @@ default set[Define] getCursorDefinitions(list[Tree] cursor, Tree(loc) _, TModel(
     return {};
 }
 
-default tuple[set[loc] defFiles, set[loc] useFiles] findOccurrenceFiles(set[Define] cursorDefs, list[Tree] cursor, Tree(loc) _, Renamer r) {
+default tuple[set[loc] defFiles, set[loc] useFiles, set[loc] newNameFiles] findOccurrenceFiles(set[Define] cursorDefs, list[Tree] cursor, str newName, Tree(loc) _, Renamer r) {
     loc f = cursor[0].src.top;
     if (any(d <- cursorDefs, f != d.defined.top)) {
         r.error(cursor[0].src, "Rename not implemented for cross-file definitions. Please overload `findOccurrenceFiles`.");
-        return <{}, {}>;
+        return <{}, {}, {}>;
     }
 
-    return <{f}, {f}>;
+    return <{f}, {f}, {}>;
```

Comment:
shouldn't we make sure that `f` might contain something of the new name?

Reply:
Good point. I'll launch a PR on https://github.com/SWAT-engineering/rascal-rename-framework/ once this is a bit stable to discuss proper defaults etc.